### PR TITLE
Improve docosaurus rendering

### PR DESCRIPTION
**Pull request by majamassarini**

### CODE REVIEW

File: docs/fedora-releases-guide.md [hunk 1]
```diff
@@ -39,22 +39,23 @@ Here are the key differences between the two:
...
 <td><p>Have a <code>.packit.yaml</code> in <b>upstream</b> repo:</p>
 
-<p>
-<code>jobs:</code><br/>
-<code>&nbsp;&nbsp;propose-downstream:</code><br/>
-<code>&nbsp;&nbsp;&nbsp;&nbsp;...</code><br/>
-</p>
+
+```yaml
+jobs
```
Comment (nforro) [inline docs/fedora-releases-guide.md:44]:
```suggestion
jobs:
```

**Review by nforro — APPROVED** [review]
GitHub renders it nicely, let's see about Docusaurus.

File: docs/fedora-releases-guide.md [hunk 2]
```diff
@@ -39,22 +39,23 @@ Here are the key differences between the two:
...
+  propose-downstream:
```
Comment (nforro) [inline docs/fedora-releases-guide.md:45]:
```suggestion
  job: propose-downstream:
```

File: docs/fedora-releases-guide.md [hunk 3]
```diff
@@ -39,22 +39,23 @@ Here are the key differences between the two:
...
 <td><p>Have a <code>.packit.yaml</code> in <b>dist-git</b> repo (main or rawhide branch):</p>
 
-<p>
-<code>jobs:</code><br/>
-<code>&nbsp;&nbsp;pull-from-upstream:</code><br/>
-<code>&nbsp;&nbsp;&nbsp;&nbsp;...</code><br/>
-</p>
+```yaml
+jobs:
+  pull-from-upstream:
```
Comment (nforro) [inline docs/fedora-releases-guide.md:56]:
```suggestion
  job: pull-from-upstream:
```

**Comment by majamassarini** [comment]
recheck

**Comment by nforro** [comment]
recheck

**Pull request merged**